akka-docs-dev/rst/java/code/docs/http/javadsl/HandlerExampleSpec.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.javadsl;
+
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.server.*;
+import akka.http.javadsl.server.values.Parameters;
+import akka.http.javadsl.server.values.PathMatchers;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+import akka.http.javadsl.testkit.TestRoute;
+import org.junit.Test;
+
+public class HandlerExampleSpec extends JUnitRouteTest {
+    @Test
+    public void testSimpleHandler() {
+        //#simple-handler-example-full
+        class TestHandler extends akka.http.javadsl.server.AllDirectives {
+            //#simple-handler
+            Handler handler = new Handler() {
```

Comment:
btw, since we now have Functions generated for `japi` in 2.3.x (I think since 2.3.11?), and bumped the streams dependency, wouldn't it make sense to drop Handler and use those `akka.japi.function.FunctionN` interfaces?